packages/codegen/src/generateEnumFile.ts

```diff
@@ -42,6 +45,10 @@ export function generateEnumFile(config: Config, enumData: EnumTableData, enumNa
         })
         .join(",")}
     };
+    
+    export const ${detailsName} = {
```

Comment:
Cool, added. I was kinda thinking about doing a `satisfies` so that maybe some of the `const`-ness of `extraColumns` / something like that would flow through, but the `details` map is already typed specifically to the `${detailsName}` type anyway.

...`keyof` is odd with enums, so had to do `${enumName}[0]` but yep, pushed up.